Assets/Scripts/HandController.cs

```diff
@@ -59,16 +65,21 @@ public class HandController : MonoBehaviour {
 
     void DeselectCard (CardController cardController) {
         selectedCards.Remove (cardController);
-        cardController.UpdateCardDisplay (0);
+        cardController.UpdateCardIndexDisplay (-1, this.canCast);
     }
 
     void UpdateAllSelectedCards (bool canCast) {
         foreach (CardController selectedCardController in selectedCards) {
-            selectedCardController.UpdateCardDisplay (RetrieveDisplayIndex (selectedCardController), canCast);
+            selectedCardController.UpdateCardIndexDisplay (RetrieveDisplayIndex (selectedCardController), canCast);
         }
     }
 
     int RetrieveDisplayIndex (CardController cardController) {
         return selectedCards.IndexOf (cardController) + 1;
     }
+
+    // compact helper to read Json file then deserialize into a Spell reference attached to a CardController
+    Spell ReadJsonToSpell (string path) {
```

Comment:
Method ordering - can we move this below `AlignCard()` so its closer to its caller?